examples/rest-xml-json/README_es.adoc

```diff
@@ -0,0 +1,404 @@
+:index-group: Unrevised
+:jbake-type: page
+:jbake-status: status=published
+= REST Simple 
+
+Definir un servicio REST es bastante fácil, simplemente agregue la anotación @Path a una
+clase luego defina en los métodos el método HTTP a usar (@GET, @POST, …).
+
+El Código
+
+== El servicio REST: @Path, @Produces, @Consumes
+
+Aquí vemos un bean que usa las anotaciones @Path, @Produces y @Consumes.
+La anotación @Path identifica la plantilla de ruta de URI a la que responde el recurso y se especifica en el nivel de clase de un recurso.
+El valor de la anotación @Path
+es una plantilla de ruta de URI parcial relativa al URI base del servidor en el que se implementa el recurso, la raíz de contexto del WAR y el patrón de URL al que
+responde el servlet auxiliar de Jersey.
+La anotación @Produces se utiliza para especificar los tipos de medios MIME o las representaciones que un recurso puede producir y enviar al cliente.
+Si @Produces se aplica en
+el nivel de clase, todos los métodos de un recurso pueden producir los tipos MIME especificados de forma predeterminada. Si se aplica a nivel de método, sobrescribir cualquier @Produces
+anotaciones aplicadas a nivel de clase.
+La anotación @Consumes se utiliza para especificar qué tipos de representaciones de medios MIME puede aceptar o consumir un recurso del cliente.
+Si se aplica @Consumes
+a nivel de clase, todos los métodos de respuesta aceptan los tipos MIME especificados de forma predeterminada. Si @Consumes se aplica a nivel de método, anula cualquier @Consumes
+anotaciones aplicadas a nivel de clase.
+
+En realidad líneas:
+
+[source,java]
+----
+@Produces({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
+@Consumes({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
+----
+
+son opcionales ya que es la configuración predeterminada. Y estas líneas pueden
+configurarse también por método si necesita ser más preciso.
+
+[source,java]
+----
+@Path("/greeting")
+@Produces({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
+@Consumes({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
+public class GreetingService {
+    @GET
+    public Response message() {
+        return new Response("Hi REST!");
+    }
+
+    @POST
+    public Response lowerCase(final Request message) {
+        return new Response(message.getValue().toLowerCase());
+    }
+}
+----
+
+== Pruebas
+
+=== Prueba del servicio JAXRS
+
+La prueba usa el ApplicationComposer de OpenEJB para hacerlo trivial.
+
+La idea es primero activar los servicios jaxrs. Esto se hace usando
+Anotación @EnableServices.
```

Comment:
"Esto se hace usando
Anotación @EnableServices." 

puede cambiar a

"Esto se hace usando la 
anotación @EnableServices."

Reply:
The correction it is maked.